### PR TITLE
fix(NcChip): adjust border radius to new design

### PR DESCRIPTION
**Pull request by marcoambrosini**
## 🏚️ Before
Screenshot 2024-08-05 at 18 32 48

## 🏡 After
Screenshot 2024-08-05 at 18 33 11

### CODE REVIEW

**Comment by susnux** [comment]
Should we really do this? The avatar is the main focus of the userbubble, at least from my point of view having it not rounded looks weird because of the now visible corners around the avatar

File: src/components/NcChip/NcChip.vue [hunk 1]
```diff
@@ -190,7 +190,7 @@ const onClose = () => {
 <style scoped lang="scss">
 .nc-chip {
 	--chip-size: 24px;
-	--chip-radius: calc(var(--chip-size) / 2);
+	--chip-radius: var(--border-radius-small);
```
Comment (susnux) [inline src/components/NcChip/NcChip.vue:193]: Does not work as this does not exist on Nextcloud 28 and 29, so the fallback to current design is needed.

**Comment by marcoambrosini** [comment]
Good point @susnux, then let's keep userbubble as is and only change chip

**Comment by ShGKme** [comment]
TBH I also like chips in pills layout more. They are small and often have rounded avatar as user bubble, for example, when they are used to select users.

**Comment by marcoambrosini** [comment]
They don't look ok when mixed up with other elements. 
Screenshot 2024-08-12 at 15 50 38
Anything else preventing this prom getting in?

**Comment by susnux** [comment]
> TBH I also like chips in pills layout more. They are small and often have rounded avatar as user bubble, for example, when they are used to select users.

Here on github it is pretty the same, GitHub is not rounded, maybe even less than our new design. Yet they use round chips for e.g. issue labels:
image

Here you can see un-rounded UI elements next to the chips:
image

**Comment by marcoambrosini** [comment]
All right, let's close

**Pull request closed without merging**